projects/storefrontapp-e2e-cypress/cypress/e2e/regression/my-account/order-history-orders-flow.e2e.cy.ts

```diff
@@ -36,8 +43,9 @@ describe('Order History with orders', () => {
   });
 });
 
-describe('Order details page', () => {
+describe('Order details page', { testIsolation: false }, () => {
   viewportContext(['mobile', 'desktop'], () => {
+    isolateTestsBefore();
```

Comment:
using isolateTestsBefore() instead of isolateTests().
getting a token error if clearing localStorage between 'it' blocks.